pkgs/applications/misc/polychromatic/default.nix

```diff
@@ -63,7 +65,7 @@ python3Packages.buildPythonApplication rec {
   ];
 
   dontWrapGapps = true;
-  dontWrapQtApps = true;
+  dontWrapQtApps = false;
```

Comment:
We want to keep this, otherwise we double wrap the binaries.

Reply:
If i have dontWrapQtApps = true, the error is present, with it false, the error is no longer present at least on my system running just xmonad, not plasma

Reply:
```suggestion
  dontWrapQtApps = true;
```